index.src.html

```diff
@@ -3728,8 +3730,8 @@
 
 : <dfn method for=VideoFrame>copyTo(|destination|, |options|)</dfn>
 :: Asynchronously copies the planes of this frame into |destination| according
-    to |options|. The format of the data is the same as this {{VideoFrame}}'s
-    {{VideoFrame/format}}.
+    to |options|. The format of the data is specified by {{VideoFrameCopyToOptions/format}},
+    if present, or the same as {{VideoFrame/format}}.
```

Comment:
Recommend replacing "or" with "otherwise".

Reply:
"or the same as" -> "otherwise"